controllers/dataprotection/type.go

```diff
@@ -29,6 +29,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	trueVal = "true"
+)
```

Comment:
move it to constant file and can be reused

Reply:
TrueVal is a constant and is used by many controller packages. Constants should only need to be defined once in a project?